.vscode/launch.json

```diff
@@ -8,7 +8,6 @@
             "name": "Attach",
             "type": "cppvsdbg",
             "request": "attach",
-            "processName": "FlightSimulator.exe",
```

Comment:
```suggestion
            "processId": "${command:pickProcess}",
```
You can use this instead.

Reply:
That is actually the default, but we can make it explicit I guess.

Reply:
https://code.visualstudio.com/docs/cpp/launch-json-reference#_additional-properties